src/Moderation/Sidebar.jsx

```diff
@@ -0,0 +1,74 @@
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border-right: 1px solid #eceef0;
+  gap: 24px;
+  width: 240px;
+  height: 100%;
+`;
+
+const Title = styled.h2`
+  font: var(text-l);
+  font-weight: 600;
+`;
+
+const MenuItem = styled.div`
```

Comment:
For the short term, let's at least make `MenuItem` a `<button>` element. That way it will be more accessible. You might need to review the styles after switching from `div` to `button`. Often times, adding `all: unset;` to your button styles can be helpful to reset any inherited styles.

Reply:
Nice! Done.